### DISCUSSION TITLE
Strange SignalR Error: Failed to start the transport 'WebSockets': Error: There was an error with the transport.

### DISCUSSION THREAD

**Discussion by ViacheslavUKR** [dotnet/aspnetcore#25479] [General]
I'm sorry, I can don't detect a problem with SignalR. I created a test for SignalR communication. I have Hub in my backend and frontend in Blazor. If I start my test from my developer computer (from Windows) all communication is working perfectly. But if I deploy the same Blazor frontend to server (OpenSuse Leap 15.1, Kestrel service, NGINX), unexpectedly the same test, the same Blazor program can not connect to the same server endpoint. Hmmmm... What I must to check? 
![Blazor-T012020_030851_1](https://user-images.githubusercontent.com/20413407/91781471-0f103980-ec03-11ea-83cd-d1928f505772.gif)

**Comment by engineering87** [comment]
Please ensure that the SSL certificate on the server is valid and correctly configured. Additionally, verify that WebSocket is available and properly enabled on the server.
To enhance reliability, consider implementing fallback mechanisms to handle cases where the WebSocket connection fails.
